contracts/AirbroCampaignFactory.sol

```diff
@@ -14,14 +14,31 @@ contract AirbroCampaignFactory is AirdropAdmin {
     address public immutable treasury;
     // protocol fee for claiming dropCampaign rewards
     uint256 public claimFee = 2_000_000_000_000_000; // 0.002 ETH
+    // protocol fee for creating dropCampaigns
+    uint256 public creatorFee = 0; // 0.000 ETH
     uint16 public claimPeriodInDays = 60;
+    bool public beta = true;
 
+    address public immutable betaAddress = 0x185310a0C79A9389e5552E338214EA86F0ef0f33;
     address public immutable erc20DropCampaign;
     address public immutable erc1155DropCampaign;
     address public immutable sb1155DropCampaign;
 
     event ClaimFeeChanged(uint256 indexed claimFee);
+    event CreatorFeeChanged(uint256 indexed creatorFee);
     event ClaimPeriodChanged(uint16 indexed claimPeriod);
+    event BetaClosed();
+
+    error NotBetaAddress();
+    error InvalidFeeAmount();
+    error FeeNotSent();
+
+    modifier duringBeta() {
```

Comment:
Do beta stuff in separate contract, and inherit in this one

Reply:
Regarding your answer "Because we don't share a PK for that account, so lets say that I want to deploy code on testnet to play with it, I need to modify files and deploy. Easier way would be to change .env" to my question about having the betaAddress as a predefined constant or a constructor param - are you saying 
a) there is a way I can import the .env var into the .sol file, so in that way everyone in their local environment can put their own betaAddress without changing the .sol file
- or - 
b) it should be a constructor param and in that way everyone can add their own betaAddress @bxmmm1

Reply:
b, .env -> constructor -> immutable var